src/helpers/approvals-satisfied.ts

```diff
@@ -80,15 +83,27 @@ export const approvalsSatisfied = async ({
 
     const numberOfRequiredReviews =
       teamOverrides?.find(({ team }) => team && entry.owners.includes(team))?.numberOfRequiredReviews ?? number_of_reviewers;
-    core.info(`Current number of approvals satisfied for ${entry.owners}: ${numberOfApprovals}`);
-    core.info(`Number of required reviews: ${numberOfRequiredReviews}`);
+    logs.push(`Current number of approvals satisfied for ${entry.owners}: ${numberOfApprovals}`);
+    logs.push(`Number of required reviews: ${numberOfRequiredReviews}`);
 
     return numberOfApprovals >= Number(numberOfRequiredReviews);
   };
 
-  core.info(`Required code owners: ${requiredCodeOwnersEntriesWithOwners.map(({ owners }) => owners).toString()}`);
+  logs.push(`Required code owners: ${requiredCodeOwnersEntriesWithOwners.map(({ owners }) => owners).toString()}`);
+  const logsJoined = logs.join('\n');
+
   const booleans = await Promise.all(requiredCodeOwnersEntriesWithOwners.map(codeOwnersEntrySatisfiesApprovals));
-  return booleans.every(Boolean);
+  const approvalsSatisfied = booleans.every(Boolean);
+
+  core.info(logsJoined);
+
+  if (!approvalsSatisfied) {
+    await createPrComment({
+      body: 'PRs must meet all required approvals before entering the merge queue.\n\n' + logsJoined
```

Comment:
This helper doesn't only run in the context of the merge queue, so we should make the message more generic. Maybe like 
```
Required approvals not satisfied: <logsJoined>
```
and we could pass the existing `body` input into this helper so the merge queue invocation can provide a merge queue-specific message too